luminance-gl/src/gl33/state.rs

```diff
@@ -56,8 +56,8 @@ pub struct GLState {
 
   // blending
   blending_state: BlendingState,
-  blending_equation: Equation,
-  blending_func: (Factor, Factor),
+  blending_equations: (Equation, Equation),
```

Comment:
I think we need to apply the same patch as suggested below: we need to be more explicit about which `Equation` applies to what (same for factors).